### DISCUSSION TITLE
How to set external links to open in new tabs?

### DISCUSSION THREAD

**Discussion by theodorechu** [marp-team/marp#63] [Q&A] [answered]
Is there a way to make external links (links to other web pages) open in new tabs? 

* The links generated by marp-cli open on the same page. 
* I tried to set `linkify: true` in the Markdown-It options for Marp Core, but it doesn't help. 

I saw that the web app at [web.marp.app](https://web.marp.app) does it like [this](https://github.com/marp-team/marp-web/blob/master/src/marp/marp.ts#L53-L59), but it seems a bit complicated. Is there an easier way?

**Comment by yhatt** [comment, marked as answer]
### For user

Just use `<a href="https://example.com/" target="_blank"></a>` with enabling HTML in your Marp tool.

### For developer

In fact, Marp Web takes the simplest way which is introduced in markdown-it documentation: https://github.com/markdown-it/markdown-it/blob/master/docs/architecture.md#renderer

Try to use [markdown-it-link-attributes](https://github.com/crookedneighbor/markdown-it-link-attributes) markdown-it plugin if you want more easy way. (NOTE: I did not yet verified compatibility with Marp)

**Reply by theodorechu** [reply]
Okay, thanks!
